i18n/en.toml

```diff
@@ -85,3 +85,16 @@ other = "Was this page helpful?"
 other = "Yes"
 [feedback_answer_no]
 other = "No"
+
+# Sign language section
+[sign_language_header]
+other = "In sign language"
+[sign_language_info]
+other = """
+**Note**:
+While each country has its own sign language we, the
+[Deaf/Hoh Working Group](https://contribute.cncf.io/accessibility/deaf-and-hard-of-hearing/),
```

Comment:
I’m not sure if we need to specify the working group. While introducing it could help raise awareness of the group and its activities, it might provide too much information for readers.

Reply:
Right, I think that it's a good opportunity to raise awareness of the WG, and also allow a user to know who to contact if they'd like to get involved. My 2 cents.

Reply:
Hi sign language team, @jtjackson @bashtheshell @dcdavisintech @Deafveloper (CC @CathPag)
Could you please review the revised note and check if it aligns with your intention?